_includes/_livestream.html

```diff
@@ -0,0 +1,74 @@
+<!-- Livestream -->
+<script>
+  document.addEventListener('DOMContentLoaded', function () {
+    let livestreamData = {{ site.data.livestreams | jsonify | strip_newlines
+  }};
+
+  if (!livestreamData || livestreamData.length === 0) {
+    return;
+  }
+
+  function isTimeInRange(startTime, endTime) {
+    let currentTime = Date.now();
+    return currentTime >= startTime && currentTime <= endTime;
+  }
+
+  let isStream = false;
+  let playerUrl = '';
+  let currentPageTitle = "{{ page.name }}";
+
+  livestreamData.forEach(function (livestream) {
+    let startTime = new Date(livestream.startTime).getTime();
+    let endTime = new Date(livestream.endTime).getTime();
+
+    let isPageMatch = livestream.pages.some(function (page) {
+      return page === currentPageTitle || page === 'Churchwide (all location pages)';
+    });
+
+    if (isPageMatch && isTimeInRange(startTime, endTime)) {
+      playerUrl = livestream.url || 'https://resi.media/cr-cinc-oh/Manifest.m3u8';
+      isStream = true;
+
+      // Refresh the page one minute after the livestream starts
```

Comment:
Since these pages are static, force refresh one minute after the start time and one minute after the end time to ensure that the stream time gets reflected.